### ISSUE TITLE
Finished with error: FileSystemException: Cannot open file, path = '/Volumes/../flutter_clean_architecture/example/ios/Runner.xcodeproj/project.pbxproj' (OS Error: No such file or directory, errno = 2)

### ISSUE THREAD

**Issue by aromal-sasidharan** [ShadyBoukhary/flutter_clean_architecture#5]
I am not able to run the project because there is no xcode project in it. What i found was Generated.config 

Please help!.

**Comment by ShadyBoukhary** [comment]
Hello,

This is a package, not an application. It's a collection of classes that you can use in your own application. It does not have a main method and therefore cannot be run as an application. 

To use it, just add it as a dependency in your `pubspec.yaml` as follows:

```dart
dependencies:
  flutter_clean_architecture: ^1.0.2
```

**Comment by aromal-sasidharan** [comment]
Ok, understood. Actually I was trying to run the example in the folder. But it has no iOS or Android source cod. Those folders just has the config files. Please help!

**Comment by ShadyBoukhary** [comment]
My apologies, some files were missing. I fixed it in #6 pull request. Use the latest version of the package and the missing files should be there.

```dart
dependencies:
  flutter_clean_architecture: ^1.0.3
```
Let me know if that works for you.
